Processes/index.rst

```diff
@@ -0,0 +1,11 @@
+Developing
+=========
+
+The electrical team is responsible for developing hardware and software for the car.
+Every team has their own repositories and how they like to do things but here we will discuss things that are common.
+.. toctree::
```

Comment:
Not sure what happened the toctree didn't generate links but instead thought it was a code block?